src/App/Resources/AppResources.resx

```diff
@@ -2840,6 +2843,9 @@ Do you want to switch to this account?</value>
   <data name="ContactSupportDescriptionLong" xml:space="preserve">
     <value>Can’t find what you are looking for? Reach out to Bitwarden support on bitwarden.com.</value>
   </data>
+  <data name="PrivacyPolicyDescriptionLong" xml:space="preserve">
+    <value>Check out our privacy policy on bitwarden.com.</value>
+  </data>
```

Comment:
For a cleanup PR, but the string value is not very long and we could put it directly onto the name key:
```xml
<data name="CheckOutOurPrivacyPolicyOnX" xml:space="preserve">
    <value>Check out our privacy policy on {0}.</value>
</data>
```
Also I'd use string interpolation to add the website. I know there are several places with string values with the website in it, but I feel links to websites should be set dynamically so they are easier to change if needed without affecting string translations.